### PR TITLE
Add responsive design for SettingsIcon component

### PR DESCRIPTION
**Pull request by banky**
See https://github.com/benawad/dogehouse/issues/1324 for more details

Desktop: 
image

Mobile: 
image

### CODE REVIEW

**Review by MadanBhandari — APPROVED** [review]

**Review by HarrisonMayotte — APPROVED** [review]

File: kibbeh/src/ui/SettingsIcon.tsx [hunk 1]
```diff
@@ -15,7 +15,9 @@ export const SettingsIcon: React.FC<SettingsIconProps> = ({
 }) => {
   return (
     <div
-      className={`w-full items-center px-4 py-2 cursor-pointer hover:bg-primary-700 ${classes}`}
+      className={`
+      w-full items-center px-4 py-2 cursor-pointer hover:bg-primary-700
```
Comment (ajmnz) [inline kibbeh/src/ui/SettingsIcon.tsx]: Should be `py-3`

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/SettingsIcon.tsx (hunk 1 above)
Comment (banky) [reply kibbeh/src/ui/SettingsIcon.tsx]: Gotcha, updated for `sm`

**Review by HarrisonMayotte — APPROVED** [review]
Appears to resolve issue from before.

**Comment by ajmnz** [comment]
@HarrisonMayotte I’m no longer reviewing PRs, request Ben’s review from now on

**Comment by HarrisonMayotte** [comment]
Sounds good @ajmnz. I'll remove you and add him instead.

**Pull request merged**